roles/dtc/common/templates/ndfc_fabric/dc_vxlan_fabric/advanced/dc_vxlan_fabric_advanced.j2

```diff
@@ -1,5 +1,5 @@
 {# Auto-generated NDFC DC VXLAN EVPN Advanced config data structure for fabric {{ vxlan.fabric.name }} #}
-  OVERLAY_MODE: cli
+  OVERLAY_MODE: {{ vxlan.global.overlay_mode | default(defaults.vxlan.global.overlay_mode) }}
```

Comment:
We should discuss if we really want to support this.  This only matters IF we are managing the entire fabric using NaC.  If we are not then this setting will not get processed.  For example if we are using --tags to only manage things other then the fabric settings